docs/helpers.md

```diff
@@ -1,6 +1,8 @@
 @property {Object.<String,can-stache.helper>} can-component.prototype.helpers helpers
 @parent can-component.deprecated
 
+@deprecated Use [can-component.prototype.ViewModel] methods instead.
```

Comment:
I understand what’s being said here, but a quick example would go a long way to helping others understand.